src/App.js

```diff
@@ -1,7 +1,26 @@
-import './App.css';
+import { Routes, Route } from 'react-router-dom';
+
+import { Provider } from 'react-redux';
+import store from './Redux/configureStore';
+
+import Header from './components/header/Header';
+import Rockets from './components/rocket/Rockets';
+import Profile from './components/profile/Profile';
+import Mission from './components/mission/Mission';
 
 function App() {
-  return <div className="App">Hello World</div>;
+  return (
+    <div>
+      <Provider store={store}>
+        <Header />
+        <Routes>
+          <Route path="/" element={<Rockets />} />
+          <Route path="/mission" element={<Mission />} />
+          <Route path="/profile" element={<Profile />} />
+        </Routes>
+      </Provider>
```

Comment:
- [ ] It would be better if we are able to run the app without any errors, although the linter checks passed the github validation, we are unable to run the app. We can fix this by carefully looking at the errors in the browser and fixing it accordingly as we can see in the screenshot below
![Screenshot from 2022-09-29 11-39-06](https://user-images.githubusercontent.com/46388113/193022765-dd9e5e45-acb0-49a2-9f6f-6281722600b1.png)